lib/model/model.go

```diff
@@ -1944,8 +1944,6 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 	if err := runner.CheckHealth(); err != nil {
 		l.Debugln("Stopping scan of folder %s due to: %s", folderCfg.Description(), err)
 		return err
-	} else if len(batch) > 0 {
```

Comment:
I don't see the point of removing this. There is still the batch sending in the loop above, this is just to handle the last, incomplete batch.

Reply:
This is for small batches, e.g. the example case of replacing a directory with a symlink/file: Without this the new symlink/file will be transmitted in a single batch, then the database is walked and the deletion below that sent in another batch. In this time, the remote device will have the problem that it should delete a directory, but can't because there is valid content.  
Also I didn't see the point of having this in the first place: Why send twice before reaching the max batch size or end of scanning?

Reply:
OK, makes sense.